torch/_inductor/cudagraph_trees.py

```diff
@@ -1792,6 +1793,40 @@ def run(self, new_inputs: List[Tensor], function_id: FunctionID):
     def set_to_running_backward(self):
         self.running_forwards_with_pending_backwards = False
 
+    def _get_cuda_graph_recorded_tensor_checker(self) -> Callable[[Tensor], bool]:
+        return (
+            self.current_node._is_cuda_graph_recorded_tensor
+            if isinstance(self.current_node, CUDAGraphNode)
+            else lambda _: False
+        )
+
+    def new_warmup_node_id(self) -> int:
+        return next(self.warmup_node_counter)
```

Comment:
Since the Warmup Nodes are temporary, I don't think we should be hashing on their ID. Why do we need a warmup ID here ?

Reply:
sounds good